### ISSUE TITLE
Python 2.7.13

### ISSUE THREAD

**Issue by jjhelmus** [conda-forge/python-feedstock#106]
A new minor release of Python 2.7 has been released, [2.7.13](https://www.python.org/downloads/release/python-2713/).  The `2.7` should be updated with this new version.

**Comment by jjhelmus** [comment]
A few notes from the release notes that should be taken into account for the build:

```
- Issue #28248: Update Windows build and OS X installers to use OpenSSL 1.0.2j.
```
and

```
- Issue #28440: No longer add /Library/Python/site-packages, the Apple-supplied
  system Python site-packages directory, to sys.path for macOS framework builds.
  The coupling between the two Python instances often caused confusion and, as
  of macOS 10.12, changes to the site-packages layout can cause pip component
  installations to fail.  This change reverts the effects introduced in 2.7.0
  by Issue #4865.  If you are using a package with both the Apple system Python
  2.7 and a user-installed Python 2.7, you will need to ensure that copies of
  the package are installed with both Python instances.
```

**Comment by jjhelmus** [comment]
closed by #128
